crates/sui-indexer-alt-framework/src/cluster.rs

```diff
@@ -175,16 +175,17 @@ mod tests {
 
     use diesel::{Insertable, QueryDsl, Queryable};
     use diesel_async::RunQueryDsl;
+    use sui_indexer_alt_framework_store_pg::pg_store::PgStore;
```

Comment:
Let's use the re-exported dep here -- also, it's suspicious to me that we are still using `crate::db::Db` below.